Makefile.include

```diff
@@ -110,6 +113,20 @@ gen-alertmanager:     # Generate Alertmanager client.
 clean_swagger:
 	find api -name '*.swagger.json' -print -delete
 
+
+ifeq ($(shell test "${SWAGGER_UI_VERSION}" = "latest" && echo 1 || echo 0), 1)
+get_swagger_version:
+override SWAGGER_UI_VERSION = $(shell curl --silent --head https://github.com/swagger-api/swagger-ui/releases/latest | grep -E '^[lL]ocation' | sed 's;^.*/;;')
```

Comment:
@artemgavrilov I presume that you wanted to avoid calls when not updating swagger, so refactored it slightly to override the version instead and moved the download into `update-swagger`